cmd/cli/commands/images_test.go

```diff
@@ -0,0 +1,373 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+func TestExtractImagePaths(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    string
+		expected []string
+	}{
+		{
+			name:     "single absolute path",
+			input:    "Describe this image /path/to/image.jpg",
+			expected: []string{"/path/to/image.jpg"},
+		},
+		{
+			name:     "multiple images",
+			input:    "Compare /path/to/first.png and /path/to/second.jpeg",
+			expected: []string{"/path/to/first.png", "/path/to/second.jpeg"},
+		},
+		{
+			name:     "relative path",
+			input:    "What's in ./photo.webp?",
+			expected: []string{"./photo.webp"},
+		},
+		{
+			name:     "Windows path",
+			input:    "Analyze C:\\Users\\photos\\pic.jpg",
+			expected: []string{"C:\\Users\\photos\\pic.jpg"},
+		},
+		{
+			name:     "no images",
+			input:    "Just a regular prompt without images",
+			expected: nil,
+		},
+		{
+			name:     "mixed case extensions",
+			input:    "Check /path/image.JPG and /path/photo.Png",
+			expected: []string{"/path/image.JPG", "/path/photo.Png"},
+		},
```

Comment:
"path with spaces" would be a useful test case. I'm terrible at evaluating regexes in my head, but I'm pretty sure we won't handle those correctly.

Reply:
Fixed in [b3bff1f](https://github.com/docker/model-runner/pull/269/commits/b3bff1f4bfab8e68a386fc81bb5a5b356fbda526)